### PR TITLE
remove debug entry from execute command contents

### PR DESCRIPTION
**Pull request by jbabazadeh**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-dc.paloaltonetworks.com/browse/XSUP-32729
## Description
This Pr fix an issue in the execute_command, in Xsoar8 when running in debug mode the server return the command result and another entry type=16 that is a debug-entry that contain the log information(file location, fileID, investigationId)
This is a BC and break customer custom script that move from xsoar6 to 8.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 1]
```diff
@@ -7711,8 +7712,7 @@ def execute_command(command, args, extract_contents=True, fail_on_error=True):
             return res
         else:
             return True, res
-
-    contents = [entry.get('Contents', {}) for entry in res]
+    contents = [entry.get('Contents', {}) for entry in res if entry['Type'] != entryTypes['debug']]
```
Comment (yuvalbenshalom) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:7715]: I think you should remove it only if its contents is empty.
and also pleas explain in the PR details why this is bing done.(that version 8 is returning a dbug entry with empty contents that breaks BC)

File: Packs/Base/ReleaseNotes/1_33_22.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### CommonServerPython
+
+- Fixed an issue in **execute_command** by remove debug entry from contents
```
Comment (yuvalbenshalom) [inline Packs/Base/ReleaseNotes/1_33_22.md:6]:
```suggestion
- Fixed an issue in **execute_command** where the command returns an unexpected empty debug entry when running on XSOAR version 8.x.
```

**Review by yuvalbenshalom — CHANGES_REQUESTED** [review]
see my comments.

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py (hunk 1 above)
Comment (jbabazadeh) [reply Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:7715]: from what I understand the contents in debug-entry is always empty, the debug entry contain the debug_file location in the file key.

**Review by yuvalbenshalom — APPROVED** [review]

File: Packs/Base/ReleaseNotes/1_33_22.md (hunk 2 above)
Comment (jbabazadeh) [reply Packs/Base/ReleaseNotes/1_33_22.md]: added.

**Pull request merged**